zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Notebook.java

```diff
@@ -1020,6 +1020,11 @@ public void removeCron(String id) {
     } catch (SchedulerException e) {
       logger.error("Can't remove quertz " + id, e);
     }
+    Note note = getNote(id);
+    if (note.isTrash()) {
+      Map<String, Object> config = note.getConfig();
+      config.remove("cron");
```

Comment:
Shouldn't we check whether it is in trash when running cron job ? Otherwise if we restore the note from trash, the cron config is lost